libs/cms-translations/src/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './lib/cms-translations.resolver'
 export * from './lib/cms-translations.service'
 export * from './lib/intl.service'
 export * from './types'
+export * from './test'
```

Comment:
Very nice test support.

Is there any way to separate this from production code so that it is not shipped to production?

Reply:
Yes, maybe just remove this export from this file and have the imports use the path to the test folders.